### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have:
- Initialised React app.
- Created a directory for my components.
- Added [React Router](https://github.com/remix-run/react-router/blob/dev/examples/basic/src/App.tsx) and set two <Route>s and <Link>s for the app's navigation (Books and Categories).
- Created a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- Created a form for adding a book (no functionality yet).
- Created a separate component for this form (with inputs for a title and an author).

- Did not use any styling as is not required at this point.

### CODE REVIEW

File: src/pages/CategoriesPage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  const { categories, message } = useSelector((state) => state.categories);
+
+  return (
+    <div>
+      <div />
+      <div><p>{categories}</p></div>
+      <div><p>{message}</p></div>
+    </div>
+  );
+};
```
Comment (Deepakdanger) [inline src/pages/CategoriesPage.js]: - [x] On the Category page, we should display only a button with the text "Check status". Kindly fix it. Also, prove the nav there so we can easily get back to the home page. Right now it's showing a blank page.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/CategoriesPage.js (hunk 1 above)
Comment (bill7pearl) [reply src/pages/CategoriesPage.js]: Done

File: src/pages/CategoriesPage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <div><p /></div>
+    <div><p /></div>
+    <button type="button">Check status</button>
+    <p>HI</p>
```
Comment (danmainah) [inline src/pages/CategoriesPage.js]: - [x] kindly remove this 'Hi' message as previously requested, the categories page should display only `check status` button.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/CategoriesPage.js (hunk 2 above)
Comment (bill7pearl) [reply src/pages/CategoriesPage.js]: Done, just forgot to delete it, However, I don't see it as a cause to not approve the project

**Review by Deepakdanger — APPROVED** [review]
Hi @bill7pearl , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**